library/general/src/modules/Directory.rb

```diff
@@ -147,7 +143,6 @@ def find_data_file(relative_path)
     publish variable: :moduledir, type: "string"
     publish variable: :scrconfdir, type: "string"
     publish variable: :desktopdir, type: "string"
-    publish variable: :icondir, type: "string"
```

Comment:
We cannot remove the `Directory.icondir` call, it is [still used by some YaST code](https://github.com/search?q=org%3Ayast+%22Directory.icondir%22&type=Code)!